doc/workflow/merge_requests.md

```diff
@@ -38,3 +38,15 @@ To check out a particular merge request:
 ```
 $ git checkout origin/merge-requests/1
 ```
+
+## Ignore whitespace changes in Merge Request diff view
+
+![MR diff](merge_requests/merge_request_diff.png)
+
+It you add `w=1` option to URL, you can see diff without whitespace changes.
```

Comment:
Does this mean adding a request parameter of w=1 like below?
https://github.com/repo1/server/compare/master...jaisharma639:master-diff?w=1
I did this but it doesn't work @yms9654